build.gradle

```diff
@@ -2,13 +2,66 @@ plugins {
 	id 'fabric-loom' version '0.2.5-SNAPSHOT'
 }
 
+repositories {
+	mavenLocal()
+	maven {
+		name = "CurseForge"
+		url = "https://minecraft.curseforge.com/api/maven"
+	}
+	maven {
+		name = "GitHub"
+		url = "https://maven.pkg.github.com"
+	}
+	maven {
+		name = "JitPack"
+		url = "https://jitpack.io"
+	}
+	maven {
+		name = "NerdHubMC"
+		url = "https://maven.abusedmaster.xyz/"
+	}
+}
+
```

Comment:
This already exists lower down in the build.gradle.

Reply:
Reorganized things to not have duplicates